src/screen/320x240x16/lcd_string.c

```diff
@@ -178,17 +176,17 @@ void close_font()
 
 u8 open_font(unsigned int idx)
 {
-    char font[20];
+    char fontname[20];
```

Comment:
Can we use tempstring?

Reply:
yes we can. but not worth it to scarify the readability as well as increase the code size.